.github/workflows/auto-review-and-pr-summary.yml

```
@@ -0,0 +1,96 @@
+          FULL_PROMPT=$(printf "$PROMPT_TEMPLATE" "$SUMMARY" "$PR_TEMPLATE")
+
+          JSON_PAYLOAD=$(jq -n \
+            --arg prompt "$FULL_PROMPT" \
+            '{contents: [{parts: [{text: $prompt}]}]}')
+
+          RESPONSE=$(curl -s -X POST "https://generativelanguage.googleapis.com/v1/models/gemini-1.5-flash:generateContent?key=${{ secrets.GEMINI_API_KEY }}" \
+            -H "Content-Type: application/json" \
+            -d "$JSON_PAYLOAD")
+
+          BODY=$(echo "$RESPONSE" | jq -r '.candidates[0].content.parts[0].text')
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)